src/Features/LanguageServer/Protocol/DefaultCapabilitiesProvider.cs

```diff
@@ -95,6 +94,16 @@ public ServerCapabilities GetCapabilities(ClientCapabilities clientCapabilities)
                 }
             };
 
+            if (!supportsVsExtensions)
+            {
+                capabilities.DiagnosticOptions = new DiagnosticOptions
+                {
+                    InterFileDependencies = true,
```

Comment:
what does this do?

Reply:
indicates that diagnostics can change in one file because of changes in other files.

Reply:
I'm guessing when we split syntax and semantic diagnostics only semantic diagnsotics would need to indicate this.